app/src/main/AndroidManifest.xml

```diff
@@ -8,14 +8,18 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Calculator">
-        <activity android:name=".MainActivity">
+        android:theme="@style/Theme.AppCompat.NoActionBar">
+        <activity android:name=".MainActivity"        android:screenOrientation="portrait">
```

Comment:
Please put android:screenOrientation in a line down of android:name

Reply:
solved!